### PR TITLE
[MIG][10.0] partner_contact_nutrition_diet

### PR DESCRIPTION
**Pull request by lasley**
* Rename manifest
* Bump versions
* Change openerp references to odoo
* Update Readme template
* Add Vegan to Readme

Depends:
- [x] #349

### CODE REVIEW

File: partner_contact_nutrition_diet/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_partner_contact_nutrition_diet" model="ir.ui.view">
+        <field name="name">Partner Contact Nutrition Diet</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_contact_nutrition.view_nutrition_information"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='nutrition']/group/group[@name='other_info']" position="inside">
+                <field name="nutrition_diet_id"/>
```
Comment (grindtildeath) [inline partner_contact_nutrition_diet/views/res_partner_view.xml]: Maybe add `options="{'no_create': True}"` ? Or add a record rule to allow creation to some group ?

File: partner_contact_nutrition_diet/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_nutrition_diet","nutrition_diet group_public","model_nutrition_diet","base.group_public",1,0,0,0
```
Comment (grindtildeath) [inline partner_contact_nutrition_diet/security/ir.model.access.csv:2]: Do we really want group_public here ?

**Comment by lasley** [comment]
@max3903 - Would you mind elaborating on the design decisions [here](https://github.com/OCA/partner-contact/pull/352#discussion_r142162970) and [here](https://github.com/OCA/partner-contact/pull/352#discussion_r142160786)? I don't want to change anything that had a good reason.

File: partner_contact_nutrition_diet/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import (
+    nutrition_diet,
+    res_partner,
+)
```
Comment (hieulucky111) [inline partner_contact_nutrition_diet/models/__init__.py:8]: Why did you put import file in tuple? just curious :)

Comment (hieulucky111) [reply partner_contact_nutrition_diet/models/__init__.py:8]: Missing empty line at the end

**Review by hieulucky111 — APPROVED** [review]
LGTM, some minor adjustments

File: partner_contact_nutrition_diet/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Nutrition Diet",
+    "summary": "Set the nutrition diet of your contacts",
+    "version": "10.0.1.0.0",
+    "category": "Health",
+    "website": "http://ursainfosystems.com",
```
Comment (NL66278) [inline partner_contact_nutrition_diet/__manifest__.py:9]: website should be https://github.com/OCA/partner-contact

File: partner_contact_nutrition_diet/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_contact_nutrition_diet/__manifest__.py:3]: Change license lines to https:

File: partner_contact_nutrition_diet/data/nutrition_diet.xml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <odoo>
```
Comment (NL66278) [inline partner_contact_nutrition_diet/data/nutrition_diet.xml:2]: Put noupdate attribute on the odoo element. User might want to change the descriptions in your records.

File: partner_contact_nutrition_diet/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
```
Comment (NL66278) [inline partner_contact_nutrition_diet/models/__init__.py:2]: no more copyright lines in init files

**Review by NL66278 — CHANGES_REQUESTED** [review]
See other comments

File: partner_contact_nutrition_diet/README.rst [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+* Go to Contacts
+* Create or select a contact
+* Go to the Nutrition tab to select their diet
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/10.0
+
+
+Known issues / Roadmap
```
Comment (nikul-serpentcs) [inline partner_contact_nutrition_diet/README.rst:29]: @lasley remove section if not content

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

File: partner_contact_nutrition_diet/README.rst (hunk 8 above)
Comment (pedrobaeza) [reply partner_contact_nutrition_diet/README.rst:29]: He is no longer working with Odoo, so this PR should be superseeded if you want it to be finished.

Comment (NL66278) [reply partner_contact_nutrition_diet/README.rst:29]: @pedrobaeza Maybe we should close all the nutrition PR's then, as they will probably not be maintained?

Comment (pedrobaeza) [reply partner_contact_nutrition_diet/README.rst:29]: What you prefer.

**Pull request closed without merging**